.gitignore

```diff
@@ -0,0 +1,3 @@
+/.idea/
```

Comment:
nit and not really relevant here, normally I'd say this should go in everybody's personal `.git/info/exclude` since it is personal for everybody what ide they use.

That being said, ignore this comment :)

Reply:
this makes sense - I've removed it from the project .gitignore. 818bb34.